Assets/_Game/Scripts/Inventory/InventoryObject.cs

```diff
@@ -0,0 +1,128 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+using UnityEngine;
+using UnityEditor;
+
+[CreateAssetMenu(fileName = "New Inventory", menuName = "DLS/Inventory")]
+public class InventoryObject : ScriptableObject, ISerializationCallbackReceiver
+{
+    public string savePath = "PlayerInventory.save";

```

Comment:
Warning: Hardcoding a save like this will only allow you to save a single inventory. If you plan on saving NPC inventories, this will break.

Reply:
Just setting the defualt path there is probably a better way to do this programmatically but since they are scriptable objects so each inventory can have their own savePath and it won't break.